library/arc4.c

```diff
@@ -48,16 +48,22 @@
 
 #if !defined(MBEDTLS_ARC4_ALT)
 
+/* Parameter validation macros */
+#define ARC4_VALIDATE_RET( cond )                                       \
+    MBEDTLS_INTERNAL_VALIDATE_RET( MBEDTLS_ERR_ARC4_BAD_INPUT_DATA, cond )
```

Comment:
The order of these macro arguments is wrong.

Reply:
Fixed.